src/api/routes/company.js

```diff
@@ -72,6 +73,30 @@ export default (app) => {
 
     });
 
+    router.get("/:companyId",
+        validators.profile,
+        (req, res, next) => companyMiddleware.canAccessProfile(req.params.companyId)(req, res, next),
+        async (req, res) => {
+            const company = await new CompanyService().findById(
+                req.params.companyId,
+                // Can be safely set to true, as the middleware takes
+                // care of validation for us
+                true,
+                req.hasAdminPrivileges
+            );
+            const offers = await new OfferService().getOffersByCompanyId(
+                req.params.companyId,
+                req.targetOwner,
+                req.hasAdminPrivileges,
+                {
+                    sort: { publishDate: "desc" },
+                    limit: CompanyConstants.offers.max_profile_visible,
+                }
```

Comment:
We should perhaps do this similarly to what's done in offer service's get, that it, pass sort and limit as explicit parameters. What do you think?

Reply:
Changing that would just be a matter of consistency, right?

Reply:
yes, behaviour is the same but, currently, you're depending on Mongoose's implementation of filters

Reply:
Oh yeah, you're totally right.

Reply:
I think we're just missing this, then I'll approve